neurovault/apps/statmaps/tasks.py

```diff
@@ -0,0 +1,145 @@
+from __future__ import absolute_import
+
+import os
+import numpy
+import pandas as pd
+import neurovault
+import pylab as plt
+import nibabel as nib
+from celery import shared_task
+from scipy.stats.stats import pearsonr
+from nilearn.plotting import plot_glass_brain
+from nilearn.image import resample_img
+from django.shortcuts import get_object_or_404
+from neurovault.apps.statmaps.models import Image, Similarity, Comparison
+
+
+@shared_task
+def generate_glassbrain_image(nifti_file,pk):
+  nifti_file = str(nifti_file)
+  png_img_name = "glass_brain_%s.png" % pk
+  png_img_path = os.path.join(os.path.split(nifti_file)[0],png_img_name)
+  glass_brain = plot_glass_brain(nifti_file)
+  glass_brain.savefig(png_img_path)
+  plt.close('all')
+
+
+@shared_task
```

Comment:
We will need to put this function in somewhere as a celery task to generate the png, yes most definitely!